homework/max-of-vector/maxOfVector.hpp

```diff
@@ -3,6 +3,10 @@
 #include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+    // auto result = std::max_element(vec.begin(), vec.end());
+    int result = vec.front();
+    for (auto element : vec) {
+        if (element > result) result = element;
```

Comment:
one command one line :)

Reply:
I thought there is some kind of "max" command but not sure which and I couldn't make it work with max_element.

Reply:
yes you can use `std::max_element()`. You just need to pass itertator to begin and end of vector and the derefenrece it.

return *std::max_element(vec.cbegin(), vec.cend());